pyproject.toml

```diff
@@ -143,9 +146,6 @@ target-version = "py310"
 allow-star-arg-any = true
 suppress-none-returning = true
 
-[tool.black]
-line-length = 120
```

Comment:
Is this the default for ruff or should we specify it?

Reply:
The default is 88. It has been specified [here](https://github.com/algorandfoundation/algokit-cli/blob/472e6799546a5439c42c7ffedc30620096638c34/pyproject.toml#L61)